ably-ios/ARTRealtimePresence.m

```diff
@@ -85,11 +121,7 @@ - (void) leaveClient:(NSString *) clientId data:(id) data cb:(ARTStatusCallback)
     msg.action = ARTPresenceLeave;
     msg.data = data;
     msg.clientId = clientId;
-    msg.connectionId = [self channel].realtime.connectionId;
-    if(!msg.clientId) {
```

Comment:
Same as above: `leaveClient` takes a non-nullable clientId as argument. The compiler will catch this. In any case, we can't be adding null checks everywhere.